0_official_module/clean_missing_data.yaml

```diff
@@ -115,12 +115,12 @@ implementation:
     - inputValue: Maximum missing value ratio
     - --cleaning-mode
     - inputValue: Cleaning mode
-    - - --replacement-value
-      - inputValue: Replacement value
-    - - --cols-with-all-missing-values
-      - inputValue: Cols with all missing values
-    - - --generate-missing-value-indicator-column
-      - inputValue: Generate missing value indicator column
```

Comment:
This is List of List, especially used in official modules for optional parameters. This use case should be supported in schema.

Reply:
Fixed